src/app/frontend/deploy/deployfromsettings.scss

```diff
@@ -14,10 +14,17 @@
 
 @import '../variables';
 
+md-input-container {
+  margin-bottom: 5px;
```

Comment:
This changes margins of all input containers for entire application. Basically, you should never have CSS rule match `md-*` selector globally. You need to create a CSS class of your own and apply it to desired elements.